Src/WitsmlExplorer.Frontend/components/Sidebar/SearchFilter.tsx

```diff
@@ -0,0 +1,58 @@
+import { Search } from "@equinor/eds-core-react";
+import { Divider } from "@material-ui/core";
```

Comment:
Would it be possible to use the `eds` Divider here? 
I tried myself without luck on first attempt but it seems like something that should be possible.